base/error.go

```diff
@@ -114,6 +116,12 @@ func ErrorAsHTTPStatus(err error) (int, string) {
 			return http.StatusBadGateway, fmt.Sprintf("%s (%s)",
 				string(unwrappedErr.Body), unwrappedErr.Status.String())
 		}
+	case *gocbcore.KvError:
+		if unwrappedErr.Code == gocbcore.StatusTooBig {
```

Comment:
Can we try just checking for gocb.ErrTooBig inside the switch on unwrappedErr above?   I think that should be sufficient, and keeps all the gocb error checking in one place.